<?xml version="1.0" encoding="UTF-8"?>
<testsuite name="s3.ind.business.impl.LoginUseCaseImplTest" tests="3" skipped="0" failures="0" errors="0" timestamp="2024-05-06T18:48:43" hostname="DESKTOP-88OFGPG" time="0.326">
  <properties/>
  <testcase name="whenCredentialsAreInvalid_thenThrowsInvalidCredentialsException()" classname="s3.ind.business.impl.LoginUseCaseImplTest" time="0.291"/>
  <testcase name="whenUserNotFound_thenThrowsInvalidCredentialsException()" classname="s3.ind.business.impl.LoginUseCaseImplTest" time="0.01"/>
  <testcase name="whenLoginIsSuccessful_thenReturnsAccessToken()" classname="s3.ind.business.impl.LoginUseCaseImplTest" time="0.02"/>
  <system-out><![CDATA[]]></system-out>
  <system-err><![CDATA[]]></system-err>
</testsuite>
